### DISCUSSION TITLE
Contribution Guidelines

### DISCUSSION THREAD

**Discussion by anonymous887** [nodejs/help#4955] [Q&A] [answered]
Hi Everyone
I have been working on Node for a Year and now, I wanted to work on it and start my open source contributions and help the community.
Can someone help me getting started with it.

Please 
Thanks

**Comment by Ayase-252** [comment, marked as answer]
Hi, thank you for your interest.

https://github.com/nodejs/node/blob/master/CONTRIBUTING.md is a good start to contribute to node. Also, https://www.nodetodo.org/ is a good reference if you want to find some idea to start contribution.

**Reply by srkds** [reply]
Nodetodo.org is a great resource ❤️
👍 Thanks for sharing.
